### PR TITLE
Add kubernetes to eden infra

### PR DESCRIPTION
**Pull request by fagomezra**
Add kubernetes to eden infra for WebApp

### CODE REVIEW

File: kubernetes/storageclass.yaml.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: eosrate-storage
```
Comment (xavier506) [inline kubernetes/storageclass.yaml.yml:4]: lets call this new storage class `eden-storage`

File: kubernetes/webapp-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      app: webapp
+  template:
+    metadata:
+      labels:
+        app: webapp
+    spec:
+      containers:
+      - image: ghcr.io/gofractally/eden-webapp:3017f7771a16864211f0f05b89ceb86fec03d21b
+        imagePullPolicy: "Always"
+        name: eosrate-webapp
```
Comment (xavier506) [inline kubernetes/webapp-deployment.yaml:20]: eosrate?

File: kubernetes/storageclass.yaml.yml (hunk 1 above)
Comment (fagomezra) [reply kubernetes/storageclass.yaml.yml]: I created a new one called eden-storage

File: kubernetes/webapp-deployment.yaml (hunk 2 above)
Comment (fagomezra) [reply kubernetes/webapp-deployment.yaml]: i changed to eden-webapp

**Pull request closed without merging**